docs/hwids.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Hardware IDs
+---
+
+## Introduction
+
+Hardware IDs are used by fwupd to identify specific hardware.
+This is useful as the device-specific identifiers may be the same for different firmware streams.
+Each hardware ID has varying levels of specificity for the hardware, for instance matching only the
+system OEM, or matching up to 8 fields including the system BIOS version.
+
+For instance, Dell and Lenovo might ship a wireless broadband modem with the same chip vendor and
+product IDs of `USB\VID_413C&PID_81D7` and although the two OEMs share the same internal device,
+the firmware may be different.
+To cover this case fwupd allows adding `hardware` requirements that mean we can deploy firmware that
+targets `USB\VID_413C&PID_81D7`, but *only for Dell* or *only for Lenovo* systems.
```

Comment:
Don't use USB for an example.  You want something that's in a machine and can't be moved.

Reply:
I've used a USB realtek webcam as a better example.

Reply:
That works too, thanks.